### PR TITLE
Port KubeAPILatencyHigh alert to 3.11

### PR DESCRIPTION
**Pull request by paulfantom**
This getting changes from https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/324 as a jsonnet patch with changed metric names (needed as openshift 3.11 uses different names).
It cannot be done by vendoring kubernetes-mixins as there is no branch for k8s 1.11 available.

/cc @openshift/openshift-team-monitoring

### CODE REVIEW

File: assets/prometheus-k8s/rules.yaml [hunk 1]
```diff
@@ -99,17 +99,22 @@ spec:
   - name: kube-apiserver.rules
     rules:
     - expr: |
-        histogram_quantile(0.99, sum(rate(apiserver_request_latencies_bucket{job="apiserver"}[5m])) without(instance, pod)) / 1e+06
+        sum(rate(apiserver_request_latencies_sum{subresource!="log",verb!~"LIST|WATCH|WATCHLIST|PROXY|CONNECT"}[5m])) without(instance, pod)
```
Comment (LiliC) [inline assets/prometheus-k8s/rules.yaml]: we have all these metrics in 3.11 already?

Comment (paulfantom) [reply assets/prometheus-k8s/rules.yaml]: yes, we do. I was testing this on CI cluster which at that time was 3.11.

**Pull request merged**